lib/html.rb

```diff
@@ -33,14 +33,24 @@ def self.run_server(system_description_store, opts)
 EOF
       end
 
+
+
       begin
         setup_output_redirection
-        Server.run!
+        begin
+          Server.run!
+        rescue Errno::EADDRINUSE
+          servefailed_error= <<-EOF.chomp
+Port #{Server.settings.port} is already in use.
+To change the port run: `#{$0} serve --port PORTNUMBER`
```

Comment:
It's likely that an old machinery server is running if the port is already in use. That's why I'd tell the user that they can stop the old server or specify a different port for the new server.